src/atom.coffee

```diff
@@ -321,18 +324,16 @@ class Atom extends Model
 
   # Extended: Invoke the given callback whenever there is an unhandled error.
   #
-  # * `callback` {Function} to be called whenever there is an unhandled error
-  #   * `event` {Object}
-  #     * `originalError` {Object} the original error object
```

Comment:
I would like to deprecate those fields for 2.0 since they lead to inconsistency with the assertion failure API. Just like we did pre-1.0, I think it's okay to make changes to the public API, so long as they don't actually break packages. No published package I could find is actually using this method, but I added the fields there just to be safe.